server/mldb_runner.cc

```diff
@@ -263,21 +263,19 @@ int main(int argc, char ** argv)
         try {
             filter_istream stream(addCredentialsFromUrl);
             vector<string> fileCredentials;
-            while (stream) {
```

Comment:
While stream checks chat no error has occurred, not if the stream has been read entirely.
Json::parse reads the entire stream.
No need for the while.